src/Microsoft.Extensions.ML/PredictionEnginePoolPolicy.cs

```diff
@@ -0,0 +1,53 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.ObjectPool;
+using Microsoft.ML;
+
+namespace Microsoft.Extensions.ML
+{
+    /// <summary>
+    /// <see cref="IPooledObjectPolicy{T}"/> for <see cref="PredictionEngine{TData, TPrediction}"/>.
```

Comment:
>IPooledObjectPolicy [](start = 19, length = 19)

I am not sure what it is given an over-simplified official document ([https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.objectpool.ipooledobjectpolicy-1?view=aspnetcore-2.2](https://docs.microsoft.com/en-us/dotnet/api/microsoft.extensions.objectpool.ipooledobjectpolicy-1?view=aspnetcore-2.2)). Could you please elaborate?

Reply:
I don't believe this class needs to be public. I have marked it as `internal`, since I don't think it will need to be used externally. @glennc - please raise any objections if I've misunderstood this class.

I have beef'd up the class summary doc here.